src/vm/codeman.h

```diff
@@ -433,11 +433,11 @@ typedef struct _HeapList
     DWORD               cBlocks;        // Number of allocations
     bool                bFull;          // Heap is considered full do not use for new allocations
     bool                bFullForJumpStubs; // Heap is considered full do not use for new allocations of jump stubs
-
-#if defined(_TARGET_AMD64_)
+#if defined(_TARGET_ARM64_)
+    BYTE                padding[2];
+#endif
```

Comment:
it seems there are only 2 callsites that need modification to typecast it to byte*